http-concat.php

```diff
@@ -10,7 +10,7 @@
 // We don't want concat running outside VIP Go environments.
 if ( true === WPCOM_IS_VIP_ENV ) {
 	// Activate concatenation
-	if ( ! isset( $_GET['concat_js'] ) || 'yes' === $_GET['concat_js'] ) {
+	if ( ! isset( $_GET['concat_js'] ) || 'no' !== $_GET['concat_js'] ) {
```

Comment:
A better option would be to move this to a function which we can then more easily test.